### ISSUE TITLE
Focus is lost for the first instance after pressing tab key for chrome://tracing page.

### ISSUE THREAD

**Issue by anniesullie** [catapult-project/catapult#1403]
**Issue by [yfulgaon...@etouch.net](https://code.google.com/u/102856445863677696217/)**
_2015-07-23T14:13:59.000Z_
_Originally opened as http://crbug.com/513267_

----

Chrome Version: 45.0.2454.15 (Official Build) Revision 3a1ba3bad4f392b9c08d4c20db31560fb377558d-refs/branch-heads/2454@{#102} 32/64 bit
OS: All (Win 7: Aero enabled)

<b>What steps will reproduce the problem?</b>
1. Launch chrome and go to chrome://tracing
2. Set cursor in the search field near View options and press tab key, observe
3. Now again press tab key and observe.

Focus is lost after pressing tab in step 2 and then focus is set in omnibox after step 3.

Focus should be in omnibox or on chevron icon after step 2.

This is  a non regression issue since the chevron icon appears on chrome://tracing page in M-44.

**Comment by anniesullie** [comment]
**Comment by [ranjit...@chromium.org](https://code.google.com/u/111464169202345430287/)**
_2015-07-23T17:47:44.000Z_

----

Untriaging it so that it gets addressed.

**Comment by anniesullie** [comment]
**Comment by [lpalm...@chromium.org](https://code.google.com/u/110518132196702918851/)**
_2015-08-31T15:03:40.000Z_

----

[triage] Dan, can you help to find an owner for this?

**Comment by anniesullie** [comment]
**Comment by [lpalm...@chromium.org](https://code.google.com/u/110518132196702918851/)**
_2015-08-31T15:13:16.000Z_

----

[triage] Dan, can you help to find an owner for this?

**Comment by anniesullie** [comment]
**Comment by [dbeam@chromium.org](https://code.google.com/u/dbeam@chromium.org/)**
_2015-08-31T17:59:36.000Z_

----

+third_party/catapult/OWNERs

**Comment by zeptonaut** [comment]
Looks like this might be a duplicate of #1356?
